pkg/translator/azurelogs/testdata/expected/audit-logs-2-expected.yaml

```diff
@@ -15,6 +15,15 @@ resourceLogs:
           - body:
               kvlistValue:
                 values:
+                  - key: operation.name
```

Comment:
Rerun `resourcelogs_to_logs_test.go` to generate all `-expected.yaml` files to make sure this PR does not introduce unexpected changes.